media-radio/cqrlog/cqrlog-2.4.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="CQRLOG is an advanced ham radio logger based on MySQL database"
+HOMEPAGE="https://www.cqrlog.com/"
+SRC_URI="https://github.com/ok2cqr/cqrlog/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-2"
+
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND=">=dev-lang/lazarus-1.8.0
+	>=dev-lang/fpc-3.0.2"
+
+RDEPEND="${DEPEND}
+	virtual/mysql[server]
+	dev-libs/atk
+	dev-libs/glib
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf
+	x11-libs/gtk+
+	x11-libs/libX11
+	x11-libs/pango"
+
+PATCHES=(
+	"${FILESDIR}/${PV}-makefile.patch"
+)
+
+pkg_postist() {
+	optfeature "This package optionally supports media-libs/hamlib for monitoring radio settings" media-libs/hamlib
```

Comment:
```suggestion
	optfeature "monitoring radio settings" media-libs/hamlib
```

Reply:
Thank you for suggestions, I've realized them through a new commit from my local repository.